tools/lib/provision.py

```diff
@@ -262,7 +264,24 @@ def main(options):
         else:
             print("No need to regenerate the test DB.")
 
-        run(["./manage.py", "compilemessages"])
+        # Check if translation files have been modified or not
+        sha1sum = hashlib.sha1()
+        paths = glob.glob('static/locale/*/LC_MESSAGES/*.po')
```

Comment:
I forgot to mention this before, but this also depends on the `translations.json` files under `static/locale`, so you should include those in the `glob`.